### PR TITLE
Use Guzzle Directly for Localise instead of Loco SDK

### PR DESCRIPTION
**Pull request by goncaloasimoes**
Loco SDK (https://github.com/loco/loco-php-sdk) hides the responses from the api, except for the body, so we can't access the headers. It also doesn't let us introduce certain headers into the requests. In order to check for changes (see https://localise.biz/help/developers/api-rate#etag), we need to read/write these headers.

Here we just change from the SDK to use Guzzle directly.

### CODE REVIEW

**Review by xipasduarte — APPROVED** [review]

File: lib/Clients/Service/Localise.php [hunk 1]
```diff
@@ -37,20 +36,31 @@ public function authenticate( array $args ) {
...
+		);
 
-		// Try to authenticate it.
-		try {
-			$result = $client->authVerify();
-			printf( "Authenticated as '%s'.\n", $result['user']['name'] );
-		} catch ( Exception $e ) {
+		$res = $client->request(
+			'GET',
+			'https://localise.biz/api/auth/verify',
```
Comment (csrui) [inline lib/Clients/Service/Localise.php:49]: Deves passar o `https://localise.biz/api/` como base_uri na instanciação do `Client`.

```suggestion
			'auth/verify',
```

File: lib/Clients/Service/Localise.php [hunk 2]
```diff
@@ -66,7 +76,29 @@ public function export( array $args ) : string {
 			throw new Exception( 'Authenticate should be called first' );
 		}
 
-		return $this->client->exportLocale( $args );
+		$url = sprintf(
+			'https://localise.biz/api/export/locale/%s.%s',
```
Comment (csrui) [inline lib/Clients/Service/Localise.php:80]: Após o base_uri actualizas este também.

```suggestion
			'export/locale/%s.%s',
```

File: lib/Clients/Service/Localise.php [hunk 3]
```diff
@@ -82,7 +114,15 @@ public function import( array $args ) {
 			throw new Exception( 'Authenticate should be called first' );
 		}
 
-		return $this->client->import( $args );
+		$url  = sprintf( 'https://localise.biz/api/import/%s', $args['ext'] );
```
Comment (csrui) [inline lib/Clients/Service/Localise.php:117]:
```suggestion
		$url  = sprintf( 'import/%s', $args['ext'] );
```

File: lib/Clients/Service/Localise.php [hunk 4]
```diff
@@ -82,7 +114,15 @@ public function import( array $args ) {
...
+		$body = $args['data'];
+		unset( $args['ext'], $args['data'] );
+
+		$url .= $this->get_query_string( $args );
```
Comment (csrui) [inline lib/Clients/Service/Localise.php:121]: Podes dispensar este método e usar o `http_build_query()`.

**Review by csrui — CHANGES_REQUESTED** [review]

**Review by csrui — APPROVED** [review]

**Pull request merged**